.github/workflows/build_oracle8.yml

```diff
@@ -0,0 +1,147 @@
+name: Oracle 8 Build
+
+on:
+  merge_group:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+  release:
+    types: [ created ]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  versions:
+    runs-on: ubuntu-latest
+    outputs:
+      antares-version: ${{steps.antares-version.outputs.result}}
+      antares-xpansion-version: ${{steps.antares-xpansion-version.outputs.result}}
+      antares-deps-version: ${{steps.antares-deps-version.outputs.result}}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Read antares-solver version
+        id: antares-version
+        uses: ./.github/actions/read-json-value
+        with:
+          path: 'antares-version.json'
+          key: 'antares_version'
+
+      - name: Read antares-xpansion version
+        id: antares-xpansion-version
+        uses: ./.github/actions/read-json-value
+        with:
+          path: 'antares-version.json'
+          key: 'antares_xpansion_version'
+
+      - name: Read antares-deps version
+        id: antares-deps-version
+        uses: ./.github/actions/read-json-value
+        with:
+          path: 'antares-version.json'
+          key: 'antares_deps_version'
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    container: 'oraclelinux:8'
+    strategy:
+      matrix:
+        xprs: [ XPRESS-ON, XPRESS-OFF ]
+    needs: [ versions ]
+    env:
+      XPRESSDIR: ${{ github.workspace }}/xpress
+      XPRESS: ${{ github.workspace }}/xpress/bin
+      XPRS_LIB_Path: ${{ github.workspace }}/xpress/lib
+      XPRESSDIR_CONTAINER: ${GITHUB_WORKSPACE}/xpress
+      XPRESS_CONTAINER: ${GITHUB_WORKSPACE}/xpress/bin
+      XPRS_LIB_Path_CONTAINER: ${GITHUB_WORKSPACE}/xpress/lib
+
+    steps:
+
+      - name: Install System
+        run: |
+          dnf install -y epel-release git cmake wget rpm-build redhat-lsb-core openmpi-devel
+          dnf install -y unzip libuuid-devel boost-test boost-devel gcc-toolset-9-toolchain zlib-devel python3-devel
+
```

Comment:
what's the purpose of python3-devel? later on line 86 you installed python3.

Reply:
devel version is required to build wheel which is required to pyinstaller